tests/_support/Step/Acceptance/Quotes.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Step\Acceptance;
+
+class Quotes extends \AcceptanceTester
```

Comment:
Similar to my comment on quotes if we could have a create quote with the line items set.